ui/src/main/java/pl/senla/hotel/SpringConfig.java

```diff
@@ -0,0 +1,14 @@
+package pl.senla.hotel;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@Configuration
+@ComponentScan("pl.senla.hotel")
+@PropertySource("application.properties")
```

Comment:
@ComponentScan
@PropertySource
@EnableJpaRepositories
are not required if you are using Spring Boot please read what goes with Spring Boot out of the box and also you can read what @SpringBootApplication hides inside

Reply:
ready